### ISSUE TITLE
The action "new' ist not allowed

### ISSUE THREAD

**Issue by ceekay73** [cdaecke/md_newsfrontend#2]
Hi,
tried to install on 9.5.3 and get the folowing error when clicking on "New" in FE:
The action "new" (controller "News") is not allowed by this plugin. Please check TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin() in your ext_localconf.php.

For testing I've tried on an 8.7.24 install where this error doesn't occur.

Also on both systems I'm also not getting list results (listView) even when adding a fe-user to a news entry manually, Debug output after repository call in NewsController shows empty query.

Would be great to have it working on 9.5.3. Thanks in advance.

**Comment by cdaecke** [comment]
I can't reproduce this error on 9.5.4. Have you tried to clear the cache?

If you still get this error, can you provide some more information?
e.g. 

- are there other extension which interfere with the news extension?
- have you installed the same extensions in your 8.7.24 system?

In list view, you will only get the news, which are created by the current logged in feUser. So every user will have its own list.

This extension is designed to work only for legged in feUsers. So make sure, that the page where you add the plugin is only accessible, when a user is logged in.

**Comment by ceekay73** [comment]
Thanks for your feedback. It's working now - solution for me was to delete the plugin content element and set it again with exacly the same settings.

**Comment by cdaecke** [comment]
Glad, to hear, that its working now!

**Comment by typo3ua** [comment]
**I see this error too...** 

I discovered if content element where `CType` was `list` and was the action "new" (controller "News") or any others actions for example "action" (controller "Action") ...and  after them to insert yours plugin then he does not work and I see this error ...
`The action "new" (controller "News") is not allowed by this plugin. Please check TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin() in your ext_localconf.php.` 
**OR** 
`An action "perUserAction" does not exist in controller Mediadreams\MdNewsfrontend\Controller\NewsController`

For example:
- 1 step - insert plugin -> Tab: Plugin -> choose `News system` -> save all.
... after this need to change plugin in the same content element
- 2 step - insert plugin -> Tab: Plugin -> choose `News frontend` -> save all.

..after this you will see error `The action "new" (controller "News") is not allowed by this plugin. Please check TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin() in your ext_localconf.php.`

If I create new content element with yours `ext: md_newsfrontend` then I do not see error

**Comment by cdaecke** [comment]
Thanks for this detailed error description @typo3ua !

The Plugin of ext:news saves some data to the field `pi_flexform`. Amongst others it sets the `switchableControllerActions` to `News-&gt;list;News-&gt;detail`, which means, that only the `listAction` and `detailAction` of the `NewsController` is allowed.

If you now switch this content element to the Plugin `News frontend` or to any other plugin (does not matter), the value in the field `pi_flexform` will not be touched if not needed. This means, that the `switchableControllerActions` are still set for the newly selected plugin and no other actions are allowed.

To sum it up: from my point of view, this is not a problem of the extension md_newsfrontend, but of TYPO3 in general.

**Comment by typo3ua** [comment]
Yes I am agree with you but how is it fix it?

Who to write about this?
